lib/engines/content_object_store/app/components/content_object_store/content_block_edition/show/confirm_summary_list_component.rb

```diff
@@ -8,7 +8,7 @@ def initialize(content_block_edition:)
   attr_reader :content_block_edition
 
   def items
-    details_items.push(confirm_item).push(date_item)
+    details_items.push(organisation_item).push(confirm_item).push(date_item)
```

Comment:
nice one!

Reply:
I've gone with spreading the array for this and the one below rather than the objects, as I feel like this is most readable?
```
  def items
    [
      *details_items,
      organisation_item,
      confirm_item,
      date_item,
    ]
  end
  ```

Reply:
Yeah, that works 👍